src/models/customers.ts

```diff
@@ -0,0 +1,24 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const Customers = sequelize.define('customers', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  full_name: {
```

Comment:
```suggestion
  fullName: {
```
let us follow camelCase as sequelize generate foreign keys on this way